### ISSUE TITLE
Cannot whitelist page

### ISSUE THREAD

**Issue by drksecret** [deanoemcke/thegreatsuspender#106]
I have this page that I want to whitelist, but it just does not seem to work:
https://www.focusatwill.com/music/#player

I tried the url, words from the url (the pages does not have a lot of actual text).

**Comment by Ashwell** [comment]
I'm using the Chrome Store version.

So I came looking at this issue because the directions for using the whitelist are very vague. It says "keywords" when really it seems like you are looking for, at the very least, a portion of the URL. There also aren't great instructions on if it is one URL per line or if the whitelist should be a comma delimited list of URLs (again "keywords of pages" doesn't make me think URL). After playing with adding items to the whitelist via the menu I figured it out.

Basically I think the whitelist section of the settings page could really benefit from some hand-holding directions, maybe something like....

====

**Whitelist**
Add the URL of each page you want to whitelist on a new line. Example:
> play.google.com/music/listen
> play.spotify.com
> mail.google.com

**Comment by deanoemcke** [comment]
@Ashwell That is a good suggestion. I agree that it's a bit vague as it is. I'll see about updating this for a future release.

**Comment by deanoemcke** [comment]
I have now implemented regex support for whitelists. See issue #197 for more details.
I have also added in more complete whitelist usage instructions as suggested in this issue.
